src/hotspot/share/oops/compressedKlass.cpp

```diff
@@ -32,9 +32,31 @@
 address CompressedKlassPointers::_base = nullptr;
 int CompressedKlassPointers::_shift = 0;
 size_t CompressedKlassPointers::_range = 0;
+// Note: initialization value is unchanged for -UseCompressedClassPointers, so
+// the bit mirroring UseCompressedClassPointers is off and matches the switch.
+uint64_t CompressedKlassPointers::_compression_info;
```

Comment:
Should we still initialize it?

```suggestion
uint64_t CompressedKlassPointers::_compression_info = 0;
```